app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    person_list = [Person(indiv["name"], indiv["age"]) for indiv in people]
```

Comment:
indiv is a bad name for wariable, use full names for naming

Reply:
I had a flake8 issue with the line being too long and it was like in one or two symbols, so I decided just too change the name. Thank u for comment I won`t do this again.